apps/masterbots.ai/components/layout/sidebar/sidebar-link.tsx

```
@@ -1,256 +1,311 @@
+				aria-expanded={isActive}
+				aria-controls={`category-${category.name}`}
+				className={cn(
+					'flex items-center p-2 w-full text-left',
+					isActive && 'bg-gray-200 dark:bg-mirage',
+					page === 'profile' && 'pl-6',
+				)}
+				onClick={(e) => {
+					e.stopPropagation()
+					handleClickCategory(e)
+				}}
+			>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)